modules/installation-user-infra-machines-advanced.adoc

```diff
@@ -176,164 +169,165 @@ $ ls $HOME/clusterconfig/
 auth  bootstrap.ign  master.ign  metadata.json  worker.ign
 ----
 
-At this point, you can use the Ignition config files as input to the ISO or PXE bare
+Now you can use the Ignition config files as input to the ISO or PXE bare
 metal installation procedures to install {op-system-first} systems.
 
 [id="installation-user-infra-machines-advanced_retaindisk_{context}"]
 === Retaining existing partitions
-For an ISO installation, you can add options to the `coreos-installer`
+For an ISO installation, you can add options to the `coreos-installer` command line
```

Comment:
*command-line

Reply:
ISG calls for the hyphen only when used as an adjective (e.g., "command-line interface")